Client/src/Utils/Theme/constants.js

```diff
@@ -15,7 +15,10 @@ const typographyLevels = {
 
 /* TODO Review color palette and semantic colors */
 const paletteColors = {
+	transparent: "#00000000",
```

Comment:
Do we nee to add a transparent color? Background color 'transparent' would solve that, right?

Reply:
We can accommodate that request. However, I was utilizing a transparent color in the `Client/src/Utils/Theme/constants.js` file on line 196. If I hadn't done that, the color would be undefined for the dark theme. Is that acceptable?